kms_log/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "env_name" {
+  description = "Environment name"
+}
+
+variable "region" {
+  default = "us-west-2"
+  description = "AWS Region"
+}
+
+variable "kmslogging_service_enabled" {
+  default = 0
+  description = "Enable KMS Logging service.  If disabled the CloudWatch rule will not be created."
+}
+
+variable "kinesis_shard_count" {
+  default = 1
+  description = "Number of shards to allocate to Kinesis data stream"
+}
+
+variable "kinesis_retention_hours" {
+  default = 24
+  description = "Number of hours to retain data in Kinesis data stream.  Max = 168"
+}
+
+variable "cloudwatch_filter_pattern" {
+  default = "[type, datetime, info, whitespace, json = *pii-encryption* || json = *password-digest*]"
```

Comment:
Have some vague concerns around accidentally filtering out data that we wanted to process. (For example, if we renamed the "pii-encryption" event to something else, we would miss out on input data on both sides of the pipeline, and possibly not notice.)

A couple options:
- Do the filtering in the lambda instead. This would allow us to be more explicit with logging to say what's happening. Downside is that we'd be paying a bunch of cost for lambda executions that do nothing except drop events.
- Add comments to identity-idp referencing all the places in these filters that must be kept in sync. (e.g. `# Don't change pii-encryption without also changing the filters for CloudWatch Logs and CloudTrail`)